activerecord/lib/active_record/fixtures.rb

```diff
@@ -962,10 +955,6 @@ def teardown_fixtures
       ActiveRecord::Base.clear_active_connections!
     end
 
-    def enlist_fixture_connections
-      ActiveRecord::Base.connection_handler.connection_pool_list.map(&:connection)
-    end
-
```

Comment:
This is API that apps can override to provide the proper set of connections to transact on, so we'll want to keep it and pass the enlisted connections to the sandbox.